app/views/declaration/additional_fiscal_references.scala.html

```diff
@@ -38,16 +38,17 @@
 @import views.components.fields.LabelledValue
 
 @import services.view.AutoCompleteItem
+@import models.requests.JourneyRequest
 
 @this(main_template: views.html.main_template)
 
-@(mode: Mode, itemId:String, form: Form[AdditionalFiscalReference], additionalReferences: Seq[AdditionalFiscalReference] = Seq.empty)(implicit request: Request[_], messages: Messages)
+@(mode: Mode, itemId:String, form: Form[AdditionalFiscalReference], additionalReferences: Seq[AdditionalFiscalReference] = Seq.empty)(implicit request: JourneyRequest[_], messages: Messages)
 
 @main_template(
     title = messages("declaration.additionalFiscalReferences.title")
 ) {
 
-    @components.back_link(routes.FiscalInformationController.displayPage(mode, itemId), mode)
+    @components.journey_back(routes.FiscalInformationController.displayPage(_, itemId))
```

Comment:
This underscore is as mode?

Reply:
journey_back - takes `Model => Call` function - here we use currying to have partial applied function